mac/keys.h

```diff
@@ -3,21 +3,29 @@
 /*                                                        :::      ::::::::   */
 /*   keys.h                                             :+:      :+:    :+:   */
 /*                                                    +:+ +:+         +:+     */
-/*   By: msousa <mlrcbsousa@gmail.com>              +#+  +:+       +#+        */
+/*   By: josantos <josantos@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2021/11/08 21:55:32 by msousa            #+#    #+#             */
-/*   Updated: 2021/11/08 21:55:57 by msousa           ###   ########.fr       */
+/*   Updated: 2022/04/29 16:41:42 by josantos         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #ifndef KEYS_H
 # define KEYS_H
 
-# define ARROW_UP	126
+# define ARROW_UP 126
 # define ARROW_DOWN 125
 # define ARROW_LEFT 123
 # define ARROW_RIGHT 124
 
+# define W_KEY 13
```

Comment:
should we use the convention of prefix instead of suffix like some other of the constants have

```c
# define KEY_A 0
# define KEY_S 1
# define KEY_D 2
```

same for space and enter
```c
# define KEY_SPACE 49
# define KEY_ENTER 36
```

Reply:
fine by me, I can change it